lib/solar.js

```diff
@@ -18,19 +23,24 @@ function _mean_anomaly(t) {
 // ecliptic longitude
 const Q0 = 280.459 * RAD;
 const Q1 = 0.98564736 * RAD;
-const L0 = 1.915 * RAD;
-const L1 = 0.020 * RAD;
+const L0 = 1.9148 * RAD;
+const L1 = 0.0200 * RAD;
+const L2 = 0.0003 * RAD;
 function _longitude(t) {
   const g = _mean_anomaly(t);
-  return Q0 + Q1 * t + L0 * Math.sin(g) + L1 * Math.sin(2 * g);
+  const sin_g = Math.sin(g);
+  const sin_2g = Math.sin(2 * g);
+  const sin_3g = (3 - 4 * sin_g * sin_g) * sin_g;
```

Comment:
This is wizardry.

Reply:
I'm pretty sure i never studied these... or maybe like 1 month ever. i looked it up though and yep, sure enough that's how you do triple angle identity! 💥

Reply:
You owe it to yourself to keep the recurrence formulae for the multiple angle formulas on your wall!